web/src/pages/index.tsx

```diff
@@ -1,56 +1,34 @@
-import { Img, Text, Heading, Box, SimpleGrid, Stack, Center, Flex } from '@chakra-ui/react'
-import { FaBook, FaChalkboardTeacher, FaUserGraduate, FaQuestion } from 'react-icons/fa'
+import { Img, Text, Heading, Box, SimpleGrid, Flex } from '@chakra-ui/react'
+import { FaBook, FaChalkboardTeacher, FaUserGraduate, FaQuestion, FaCalendarWeek } from 'react-icons/fa'
 import { PageScaffold } from '../components/PageScaffold'
 import WithAuth, { WithAuthProps } from '../components/withAuth'
 import { LinkBoxHomePage } from '../components/LinkBoxHomePage'
 import React from 'react'
 import source from '../images/Logo_GeMont.png'
 
 const Index: React.FC<WithAuthProps> = ({ self }) => {
-  const getTeacherLinkBoxes = () => {
+  const arrayOfLinkBoxes = [
+    { icon: FaChalkboardTeacher, url: '/tutorium', text: 'Tutorium Management' },
+    { icon: FaUserGraduate, url: '/student', text: 'Schüler Management' },
+    { icon: FaBook, url: '/absence', text: 'Abwesenheit buchen' },
+    (self.role === 'COORDINATOR')
+      ? { icon: FaCalendarWeek, url: '/semester', text: 'Zeitspanne erstellen' }
+      : { icon: undefined, url: undefined, text: undefined },
```

Comment:
Finde ich ungünstig, können wir es vielleicht so machen, dass der Array wirklich nur die Elemente hat die er braucht? Mein Vorschlag wäre das wir das entweder so machen wie in der PageScaffold, also dass du für jeden Eintrag eine Role mit definierst und dann dementsprechend abfragst, oder, dass du die Grundliste deklarierst und dann mit <liste>.push() je nachdem welche Rolle der Nutzer hat die unterschiedlichen Einträge dazufügst.

Reply:
Okay, machen wir so. Ich setze mich heute Abend nochmal dran;).

Reply:
Ich habe ein Problem mit dem Backend:/. @jonaslanglotz Wollen wir uns das gemeinsam anschauen?

Reply:
Aber eventuell funktioniert es ja bei dir?

Reply:
Wir hatten zwischenzeitlich ein Problem aufgrund einer kaputten PR die gemerged wurde. Du kannst mal schauen ob es hilft den aktuellen Stand von dev in deine Branch zu mergen.